src/kOS/Utilities/VesselUtils.cs

```diff
@@ -121,13 +121,35 @@ public static double GetMaxThrust(Vessel vessel)
                     if (enginefx != null)
                     {
                         if (!enginefx.isOperational) continue;
-                        thrust += enginefx.maxThrust;
+                        float flowMod = (float)(engine.part.atmDensity / 1.225f);
```

Comment:
I never bothered to change the variable name for `flowMod`, but really it should be called `atmMod` or some such for better readability in the future.